### PR TITLE
KOGITO-9817: Add the new SonataFlow webapp to the swf builder deployment

### PR DESCRIPTION
**Pull request by caponetto**
- Remove `serverless-logic-web-tools-swf-deployment-webapp` package
- Add the new webapp to `serverless-logic-web-tools-swf-builder-image`
- Rename data JSON file to `sonataflow-deployment-webapp-data.json`
- Rename the webapp package name to `sonataflow-deployment-webapp`

### CODE REVIEW

File: packages/sonataflow-deployment-webapp/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "sonataflow-deploy-webapp",
+  "name": "@kie-tools/sonataflow-deployment-webapp",
```
Comment (fantonangeli) [inline packages/sonataflow-deployment-webapp/package.json]: Renaming the package with the "@kie-tools/" prefix gves an error in the consumer app's pom.xml. Eg. using `@kie-tools/dashbuilder-client` as a dependency:
`[ERROR] 'dependencies.dependency.artifactId' for org.webjars.npm:@kie-tools/dashbuilder-client:jar with value '@kie-tools/dashbuilder-client' does not match a valid id pattern.`

File: packages/serverless-logic-web-tools-swf-builder-image/package.json [hunk 2]
```diff
@@ -19,17 +19,15 @@
...
     "image:podman:build": "run-script-if --bool $([ $(command -v podman) ] && echo true || echo false) --then \"podman build --ulimit nofile=5000:5000 $(echo $(build-env swfBuilderImage.buildTags) | xargs printf -- \"-t $(build-env swfBuilderImageEnv.registry)/$(build-env swfBuilderImageEnv.account)/$(build-env swfBuilderImageEnv.name):%s\n\" | xargs echo) --build-arg KOGITO_IMAGE_TAG=$(build-env swfBuilderImage.kogitoImageTag) -f Containerfile\" --else \"echo Podman not found, skipping image build.\""
   },
   "devDependencies": {
-    "@kie-tools/dashbuilder-swf-monitoring-dashboard": "workspace:*",
     "@kie-tools/root-env": "workspace:*",
     "@kie-tools/serverless-logic-web-tools-swf-builder-image-env": "workspace:*",
-    "@kie-tools/serverless-logic-web-tools-swf-deployment-webapp": "workspace:*",
+    "@kie-tools/sonataflow-deployment-webapp": "workspace:*",
```
Comment (fantonangeli) [inline packages/serverless-logic-web-tools-swf-builder-image/package.json]: We need to remove `@kie-tools/` prefix if my previous comment is accepted.

**Review by fantonangeli — COMMENTED** [review]
Thank you very much for your PR

File: packages/sonataflow-deployment-webapp/package.json (hunk 1 above)
Comment (caponetto) [reply packages/sonataflow-deployment-webapp/package.json]: Good catch! See 7130a3b4ea6955c8c7aa3440ca22a661ef655903

**Review by fantonangeli — APPROVED** [review]

**Pull request merged**